services/wallet/commands.go

```diff
@@ -731,12 +732,6 @@ func (c *controlCommand) Run(parent context.Context) error {
 	if target.Cmp(zero) <= 0 {
 		target = zero
 	}
-	ctx, cancel = context.WithTimeout(parent, 3*time.Second)
```

Comment:
This request is needed only if watching of new block is enables, thus moved to `if c.watchNewBlocks { ... }` block